.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a bug report to help us improve NFCGate
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**App Information\***
+Screenshot of "status" page
+
+**Device Information\* (please complete the following information)**
+ Device: [e.g. Google Pixel 7]  
```

Comment:
There may be a superfluous space at the beginning of this line

Reply:
Thanks, fixed!